sql/item_mods_pet.sql

```diff
@@ -128,12 +128,12 @@ INSERT INTO `item_mods_pet` VALUES (11118,25,15);     -- Accuracy +15
 -- -------------------------------------------------------
 -- Caller's Spats +2
 -- -------------------------------------------------------
-INSERT INTO `item_mods_pet` VALUES (11138,345,500);   -- TP Bonus +500
+INSERT INTO `item_mods_pet` VALUES (11138,345,500);       -- TP Bonus +500
```

Comment:
Placing this here will affect any pet not just avatars. The item help-text indicates it should only effect avatars and not any other pet type like a smn/bst charmed pet, probably not affect smn spirits, etc.

I haven't looks but if you have any other type specific one sin the mods in this pr, need to take'em out.